.github/actions/bionic/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Do not modify this file directly.
+# This file is automatically generated from packaging/generate_actions.py
+FROM ubuntu:bionic
+
+LABEL maintainer "Hiroki Kumazaki <kumagi@google.com>"
+
+RUN grep -q "# deb-src" /etc/apt/sources.list && sed -Ei 's/^# deb-src /deb-src /' /etc/apt/sources.list || \
+    cat /etc/apt/sources.list | grep "^deb " | sed -e "s/^deb /deb-src /" | sponge -a /etc/apt/sources.list
```

Comment:
You can write this more concisely as (useless use of `cat` and `grep`):

```
sed -e "s/^deb /deb-src /" < /etc/apt/sources.list | sponge -a /etc/apt/sources.list
```

Reply:
Sounds good. Done.